runservers

```diff
@@ -292,6 +292,20 @@ tp_blip() {
     echo "Blip started at http://localhost:$PORT"
 }
 
+# platform
+tp_platform() {
+    echo "Starting platform"
+    export GOPATH=/tidepool/platform
+    cd $GOPATH/src/github.com/tidepool-org/platform
+    . ./.env
+    _bin/dataservices/dataservices >> /tidepool/platform_server.log 2>&1 &
```

Comment:
We'll also need to start `userservices` at some point. It starts to be necessary with the delete user (and all associated data) functionality. So really, we need a `tp_platform_dataservices` and `tp_platform_userservices`, I think.